### PR TITLE
FIX Allow double digit minor version numbers

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-admin/issues/1274

Also removes showing the 'CWP version'

The cwp-core template overides the core LeftAndMain_MenuToggle.ss template

cwp-core already requires admin ^1.10 requirement so the new `CMSMinorVersionNumber()` method is available

### CODE REVIEW

File: templates/SilverStripe/Admin/Includes/LeftAndMain_MenuToggle.ss [hunk 1]
```diff
@@ -4,15 +4,10 @@
         <span class="cms-help__logo font-icon-silverstripe"></span>
```
Comment (maxime-rainville) [inline templates/SilverStripe/Admin/Includes/LeftAndMain_MenuToggle.ss]: This template has zero value if you don't insert the CWP version in here ... just delete.

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
You need to update 
- Deprecate [CWPVersionExtension](https://github.com/silverstripe/cwp-core/blob/2/src/Extension/CWPVersionExtension.php)
- Delete [version.yml](https://github.com/silverstripe/cwp-core/blob/2/_config/version.yml) to disable the extension
- Delete those line https://github.com/silverstripe/cwp-core/blob/db85822ba6cef92cb584e1c4b192fea97e723226/css/custom.css#L31-L47

File: templates/SilverStripe/Admin/Includes/LeftAndMain_MenuToggle.ss (hunk 1 above)
Comment (emteknetnz) [reply templates/SilverStripe/Admin/Includes/LeftAndMain_MenuToggle.ss]: Deleted

**Review by maxime-rainville — COMMENTED** [review]
Sorry ... meant to approve before merging.

**Pull request merged**